src/components/message-box/MessageBox.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Box, {COLOR} from '../box/Box';
+import Icon, {TYPE as iconTypes, ICON_COLOR} from '../icons/Icon';
+
+export {COLOR};
+
+const MessageBox = ({children, onClose, ...props}) => {
+  const boxClass = classNames({
+    'sg-box--with-close': onClose
+  });
+
+  return (
+    <div className={'sg-message-box'}>
```

Comment:
I think `className` and `styles` attributes should we passed to this div instead of `Box`